src/assets/images/avatar.svg

```diff
@@ -5,7 +5,7 @@
     <desc>Created with Sketch.</desc>
     <defs></defs>
     <g id="Search" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
-        <g sketch:type="MSArtboardGroup" transform="translate(-307.000000, -310.000000)" id="Search-results" fill="#6E6E6E">
+        <g sketch:type="MSArtboardGroup" transform="translate(-307.000000, -310.000000)" id="Search-results">
```

Comment:
the avatars are on the*CDN so we have to hard code the color, because you can't change the color with a `.svg` in a `<img>` tag

Reply:
@viktorsmari What about if we return null and we leave at the client side what default avatar picture to use? https://github.com/fablabbcn/smartcitizen-api/blob/b8e1e8c6b41f6d8d13f040c129f2fcd151ff2ecd/app/models/user.rb#L60